docker/nginx.Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM nginx:1.15-alpine
 
 WORKDIR /app/public
 
-COPY nginx.conf /etc/nginx/nginx.conf
+COPY ./docker/nginx.conf /etc/nginx/nginx.conf
```

Comment:
the context is set to `./docker` when building the image, so the path `./nginx.conf` is correct with the current `docker-compose.yml` file